src/utilities/specs/avatar.specs.js

```diff
@@ -13,14 +13,14 @@ const avatarSpec = createSpec({
         id: prop.id,
         name: prop.name,
         firstName: prop.firstName,
-        image: prop.image,
+        // image: prop.image,
```

Comment:
Leftover comment?

Reply:
Was waiting for the button PR to be merged (done now), avatar images stopped working and PL switched to another service in that PR 👍